$MirOS: ports/www/firesomething/patches/patch-xpcom_io_nsNativeCharsetUtils_cpp,v 1.3 2008/05/21 06:31:05 bsiegert Exp $
$OpenBSD: patch-xpcom_io_nsNativeCharsetUtils_cpp,v 1.3 2005/03/02 19:30:18 naddy Exp $
--- xpcom/io/nsNativeCharsetUtils.cpp.orig	2007-07-08 09:08:51.000000000 +0200
+++ xpcom/io/nsNativeCharsetUtils.cpp	2008-08-24 14:52:53.000000000 +0200
@@ -334,7 +334,11 @@ nsNativeCharsetConverter::LazyInit()
 {
     const char  *blank_list[] = { "", NULL };
     const char **native_charset_list = blank_list;
+#ifdef CODESET
     const char  *native_charset = nl_langinfo(CODESET);
+#else
+    const char  *native_charset = nsnull;
+#endif
     if (native_charset == nsnull) {
         NS_ERROR("native charset is unknown");
         // fallback to ISO-8859-1
